### PR TITLE
feat: add check for release notes in PR

### PR DESCRIPTION
**Pull request by mfocko**

### CODE REVIEW

File: workflows/check-release-notes.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    steps:
+      - name: Check if there are any release release notes and placeholder
+        env:
+          PR_DESCRIPTION: ${{ inputs.description }}
+          NOTES_PLACEHOLDER: ${{ inputs.placeholder }}
+        shell: python
+        run: |
+          import os
+
+          RELEASE_NOTES_TAG = "RELEASE NOTES"
```
Comment (lbarcziova) [inline workflows/check-release-notes.yml:26]: why not directly include the ` BEGIN`?

**Review by lbarcziova — APPROVED** [review]
nice improvement!

(maybe it's just me, but it would be useful if it could detect the TODO/TBD which I like to use and sometimes forget to replace :D)

File: workflows/check-release-notes.yml (hunk 1 above)
Comment (mfocko) [reply workflows/check-release-notes.yml:26]: taken it from the `prepare-release`, where we have just this part in a constant (that could be changed, if need be) and `BEGIN` is included below…

**Comment by mfocko** [comment]
> nice improvement!
> 
> (maybe it's just me, but it would be useful if it could detect the TODO/TBD which I like to use and sometimes forget to replace :D)

I was trying to avoid regexes as it was complicating stuff more than solving :D though I think it should be doable :eyes:

**Pull request merged**